src/app/shared/pipes/split-string-to-html.pipe.ts

```diff
@@ -0,0 +1,16 @@
+import { Pipe, PipeTransform } from "@angular/core";
+import { SafeHtml } from "@angular/platform-browser";
+@Pipe({
+  name: "splitStringToHTML"
+})
+export class SplitStringToHTMLPipe implements PipeTransform {
+  transform(value: string, delimiter: string = "|"): SafeHtml {
+    if (value && value.includes(delimiter)) {
+      return value
+        .split(delimiter)
+        .map((item: string) => `<span class='split-item'>${item}</span>`)
+        .join(" ");
```

Comment:
The delimiter from backend actually is " | " which contains a leading and a trailing whitespace, I'm a little wondering if we need to trim them.

Reply:
The output will have redundant space  `foo | bar => <span>foo </span><span> bar</span>` From a presentation perspective this extra whitespace has no impact as it is ignored when rendered in browser. Alternatively, the pipe can take any delimiter so I can pass ` | ` instead.